src/utils/jwtConfiguration.ts

```diff
@@ -0,0 +1,13 @@
+import jwt from 'jsonwebtoken'
+
+export const generateToken = (data: object) => {
+  const token = jwt.sign(data, process.env.JWT_SECRET || 'USER-AUTH', {
+    expiresIn: process.env.JWT_EXPIRE_TIME || 300000,
+  })
+  return token
+}
+
+export const decodeToken = (token: string) => {
+  const verify = jwt.verify(token, process.env.JWT_SECRET || 'USER-AUTH')
+  return verify
```

Comment:
```suggestion
  return  jwt.verify(token, JWT_SECRET )
```
can return jwt.verify directly without needing verify as it not being used